### ISSUE TITLE
Shortcode overwriting not working 

### ISSUE THREAD

**Issue by jamieisboss** [bassjobsen/woocommerce-twitterbootstrap#17]
I am attempting to use the shortcodes [featured_products] and [recent_products] in my theme but they are not showing any of the bootstrap classes and look exactly as they would without the plugin. I've also tried adding content_product_template as in one of thr examples in the readme, but to no avail, a snippet of my code is below:

`<?php echo do_shortcode('[featured_products columns="4" per_page="4" content_product_template="bs-content-product-info"]'); ?>`

I am currently running WooCommerce 2.2.4 (most recent version) which I've noticed isn't the supported version in the readme, and might well be the reason this isn't working now. Is there anything I can do to get this going?

Thanks

Jamie

**Comment by lemoslincoln** [comment]
Hey @jamieisboss , can you do this work?

**Comment by jamieisboss** [comment]
In the end i just wrote some functions to display the content the way we wanted to here's a gist of it:

https://gist.github.com/jamieisboss/a4a3f4e40a663dc04dff

I actually started rewriting this plugin from the ground up but got completeley sidetracked. I think I'll finish that off soon because i want it to integrate with woocommerce the way they suggest and i'll be wanting to move to bootstrap 4.

Looking at the code now though i actualy think i know why it stopped working and how to fix it, gimme a sec...

**Comment by jamieisboss** [comment]
Yeah, a sec, submitted a pull request which i think should fix it, cant tst myself though so ... dont know, if its not accepted i'll build a little test copy and test properly en ive got a bit more time. I forgot how complicated changing the woocommerce  behaviour  is.

**Comment by lemoslincoln** [comment]
Ok.. thx for share.
I will use the functions atm..
